sunpy/map/mapsequence.py

```diff
@@ -345,3 +345,9 @@ def all_meta(self):
         Return all the meta objects as a list.
         """
         return [m.meta for m in self.maps]
+
+    def derotate(self):
+        """
+         Return derotated layers in the MapSequence
+        """
+        return self._derotate()
```

Comment:
Why was this function created?

Reply:
I was reading somewhere that internal functions should not be directly called outside the class, so created this.

Reply:
But this function isn't called that I could see?

Reply:
It is called in the ``test_mapsequence.py``
For testing the raising of the error, I did not want call the ``_derotate()``outside the class hence created it. 
Would you rather have me call ``_derotate()`` directly for the test?

Reply:
Well the init of the class calls the underscore method.I don't see the need for this new function. Unless there is that I'm not aware of?

Reply:
Hmm. I could directly make the object with the ``derorate`` parameter set as ``True``. Would that be fine?

Reply:
Sorry, I don't follow.

Reply:
It was dumb on my part. Rather than make a new function, I have changed the test. In the test, I am making the object with the ``derotate`` set as true, so the ``NotImplementedError`` getting successfully raised is tested.